sources/packages/web/src/components/common/applicationTracker/Completed.vue

```diff
@@ -17,6 +17,15 @@
     background-color="warning-bg"
     content="StudentAid BC is reviewing your requested change. If your requested change is approved, your application will be re-evaluated with a new assessment below."
   />
+  <!-- Student request an application change (after COE) - change in progress -->
```

Comment:
Asking for my understanding, the order of the banner is something confirmed with business? e.g. block funding banner still has the precedence.

Reply:
It was not checked with the business. On second thought, I will move it all the way to the top and raise the question to them.
In my understanding, as a "parallel" process "outside" the current application version, displaying it all the way to the top would make sense.